analysis/source/descriptive/descriptive.py

```diff
@@ -3,14 +3,14 @@
 import matplotlib.pyplot as plt
 
 def main():
-    PATHS =  yaml.load(open('config_global.yaml', 'rU'))
-    data  = np.genfromtxt('%s/data.txt' % PATHS['build']['prepare_data'], skip_header = 1)
+    paths_global =  yaml.load(open('config_global.yaml', 'rU'))
+    data  = np.genfromtxt('%s/data.txt' % paths_global['build']['prepare_data'], skip_header = 1)
 
-    with open('%s/table.txt' % PATHS['build']['descriptive'], 'w') as f:
+    with open('%s/table.txt' % paths_global['build']['descriptive'], 'w') as f:
         f.write('<tab:table>\n')
         f.write('%s\n%.3f\n%d\n%d' % (np.mean(data), np.std(data, ddof = 1), np.max(data), np.min(data)))
 
     plt.hist(data)
-    plt.savefig('%s/plot.eps' % PATHS['build']['descriptive'])
+    plt.savefig('%s/plot.eps' % paths_global['build']['descriptive'])
 

```

Comment:
I see the issue with carrying around two `paths` variables. My instinct is to write a wrapper function `read_config()` that read both of these into a `path` variable or whatever it's called and then have keys for `global` and `user`.

Reply:
This work but it means that the PATHS variables we load in the scripts have a different structure than the PATHS environment variable we use in SCons.